src/app/containers/ATIAnalytics/params/article/index.test.jsx

```diff
@@ -48,79 +58,23 @@ describe('ArticleAtiParams', () => {
     previousPath: '/previous-path',
     service: 'news',
     statusCode: 200,
+    atiAnalyticsAppName: 'news',
+    atiAnalyticsProducerId: 0,
   };
 
   describe('atiPageViewParams is called ', () => {
     afterEach(() => {
       jest.clearAllMocks();
     });
 
-    it('should call atiPageViewParams with the params from the Contexts', () => {
-      const mock = jest.fn().mockReturnValue('key=value&key2=value2');
-      atiPageViewParams.default = mock;
-
+    it('should call buildArticleATIUrl exactly once', () => {
       renderer.create(Component(newsServiceContextStub, requestContextStub));
 
-      expect(mock).toHaveBeenCalledTimes(1);
-      expect(mock).toHaveBeenCalledWith({
```

Comment:
Wheres this gone?

Reply:
https://github.com/bbc/simorgh/pull/3559/files#diff-df1a49fb9b8db525415a9fddb5b326f3

Since the buildArticleATIUrl is already tested, we should only test that it's called, and called with the right arguments.